telegraf.conf

```
@@ -36,3 +36,12 @@
   data_format = "influx"
   interval    = "10m"
   timeout     = "10s"
+
+# Wallet balance monitoring
+[[inputs.exec]]
+  commands = [
+    "/opt/probes/wallet-balance.sh calibration ${CALIBRATION_FILBEAM_CONTROLLER_WALLET}",
+  ]
```

We will need to monitor the wallet balance of the mainnet controller wallet, too. Since FWSS v1.0.0 was already deployed to the mainnet, I propose to include the mainnet setup from the start.

```suggestion
  commands = [
    "/opt/probes/wallet-balance.sh calibration ${CALIBRATION_FILBEAM_CONTROLLER_WALLET}",
    "/opt/probes/wallet-balance.sh mainnet ${MAINNET_FILBEAM_CONTROLLER_WALLET}",
  ]
```

---

I had no idea that it was deployed to mainnet 👀